docs/release-notes.md

```diff
@@ -17,9 +17,24 @@
 
 # Release notes
 
+## Trino Gateway 4 (29 Nov 2023)
+
+[gateway-ha-4-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/4/gateway-ha-4-jar-with-dependencies.jar)
+
+* Add support for configuring additional whitelisted URL paths. (https://github.com/trinodb/trino-gateway/pull/63)
+* Improve flexibility of user and group name matching for authorization. (https://github.com/trinodb/trino-gateway/pull/32)
+* Add support to use external URL for backend creation. (https://github.com/trinodb/trino-gateway/pull/76)
+* Enable configuration of HTTP header size. (https://github.com/trinodb/trino-gateway/pull/67)
+* Automatically set JDBC configuration parameter for TLS when connecting to
+  clusters for monitoring. (https://github.com/trinodb/trino-gateway/pull/71)
+* Modernize application and remove potential for undiscovered bugs and security
+  issues with update of many core dependencies. (https://github.com/trinodb/trino-gateway/pull/59)
+* Ignore unhandled OIDC properties without failure. (https://github.com/trinodb/trino-gateway/pull/69)
```

Comment:
Either "Ignore unhandled OIDC properties" or "Avoid failure when unhandled OIDC properties are present".

However, I'm not entirely clear what "unhandled" means in this context. Unsupported? Something else?

Reply:
unhandled by the gateway but must be passed through to the target cluster .. before this it would fail and not pass on ..